src/components/Menu/Menu.module.css

```diff
@@ -28,8 +28,20 @@
   font-weight: var(--eds-font-weight-light);
 }
 
-/* Needed to create higher specificity than ClickableStyle. */
-.menu__button--icon {
+.menu__plain-button {
+  min-height: 42px;
+  min-width: 42px;
+}
+.menu__plain-button:focus-visible {
+  @mixin focus;
+}
+@supports not selector(:focus-visible) {
+  .menu__plain-button:focus {
+    @mixin focus;
+  }
+}
```

Comment:
Just some accessibility styling for the plain button

Reply:
@jinlee93 the designs *should* apply the accessible size, and specifying a minimum will be a visual hint that something is off if the input is too small.

We might also apply some centering in this case?

Reply:
I think it's centered by default 
![image](https://user-images.githubusercontent.com/86632227/222541005-931ae97b-4ad1-4c43-9bec-59a5c2042b3b.png)

Hesitant to add any more styling on top since TW classes don't override EDS styles in usage in edu-stack